js/templates/modals/settings/general.html

```diff
@@ -128,6 +128,23 @@ <h2 class="h3 clrT"><%= ob.polyT('settings.generalTab.sectionName') %></h2>
           </div>
         </div>
       </div>
+      <div class="flexRow gutterH TODO">
```

Comment:
This is hidden since I wrote the code then we decided not to include it. When/if another verified moderator provider exists we could turn this back on.